src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+import './TodoList.scss';
+
+type Props = {
+  preparedTodos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ preparedTodos }) => (
+  <div className="todolist">
+    {preparedTodos.map(todo => (
+      <div className="todoList" key={todo.id}>
```

Comment:
```suggestion
  <ul className="todolist">
    {preparedTodos.map(todo => (
      <li className="todoList" key={todo.id}>
```
Список виглядає більш доречним тут